ddtrace/contrib/internal/google_generativeai/patch.py

```diff
@@ -33,6 +34,10 @@ def get_version():
     return getattr(genai, "__version__", "")
 
 
+def _supported_versions() -> Dict[str, str]:
+    return {"google.generativeai": ">=0.7.0"}
```

Comment:
an aside - it looks like we missed updating this in `docs/index.rst` in the other PR (it still says `0.7.2`)